app/helpers/menu_helper.rb

```diff
@@ -83,4 +83,8 @@ def forcibly_denied_switch?(switch)
     account = current_account.provider? ? current_account : current_account.provider_account
     account.settings.public_send(switch).is_a?(Settings::SwitchDenied)
   end
+
+  def api_selector_services
+    @api_selector_services ||= site_account.provider? ? current_user.accessible_services.includes(:account) : Service.none
```

Comment:
`current_user` is not always defined. In some pages where you skip the login. Better to be safe

Reply:
right 👍